### ISSUE TITLE
JSON Output: First message duplicated as plain text

### ISSUE THREAD

**Issue by Skrupellos** [validator/validator#451]
When using releases 17.1.0 and 17.0.1 and an JSON formatted output, the first message is also printed to stdout as plain text. See the following example for further details.

The input file `test.xhtml`:
```xml
<fail>
```

The console output:
```
# java -jar vnu.jar --format json test.xhtml
"file:test.xhtml":-1:-1: warning: premature end of file (found “[EOF]”)
{"messages":[{"type":"error","url":"file:test.xhtml","lastLine":1,"lastColumn":6,"firstColumn":1,"message":"Element “fail” not allowed in this context. (Suppressing further errors from this subtree.)","extract":"<fail>","hiliteStart":0,"hiliteLength":6},{"type":"error","url":"file:test.xhtml","lastLine":1,"lastColumn":6,"subType":"fatal","message":"premature end of file (found “[EOF]”)"}]}
```

The second line (first line of output) should not be there.

**Comment by sideshowbarker** [comment]
Thanks for catching this. The fix will be in the next release.
